tensor2tensor/data_generators/mathematical_language_understanding.py

```diff
@@ -0,0 +1,104 @@
+# coding=utf-8
+# Copyright 2018 Artit Wangperawong artitw@gmail.com
```

Comment:
Need to replace this line by this: (sorry, legal department insists)
# Copyright 2018 The Tensor2Tensor Authors.

Can we do that? (And please add yourself to AUTHORS too!)

Reply:
Looks good, I confirm. Thanks, @lukaszkaiser ~